swift/apple/FirezoneKit/Sources/FirezoneKit/Models/WebAuthSession.swift

```diff
@@ -15,7 +15,7 @@ import AuthenticationServices
 struct WebAuthSession {
   private static let scheme = "firezone-fd0020211111"
 
-  static func signIn(store: Store) async {
+  static func signIn(store: Store) {
```

Comment:
The ASWebAuthenticationSession itself opens and manages the WebView so we can return immediately here.

State will be updated in our `store.signIn` callback later on.